### PR TITLE
Add streaming logs from device agents

### PR DESCRIPTION
**Pull request by hardillb**
## Description
This requires MQTT device comms

Dependent on https://github.com/flowforge/flowforge-device-agent/pull/66

## Related Issue(s)

https://github.com/flowforge/flowforge/issues/1820

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/components/DeviceLog.vue:53]:
```suggestion
```

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/components/DeviceLog.vue:60]:
```suggestion
```

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/components/DeviceLog.vue:71]:
```suggestion
```

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/components/DeviceLog.vue:76]:
```suggestion
```

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/components/DeviceLog.vue:94]:
```suggestion
```

File: frontend/src/pages/device/index.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/index.vue:111]:
```suggestion
```

File: frontend/src/pages/device/Logs.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/Logs.vue:6]:
```suggestion
```

Should remove this?

File: frontend/src/api/devices.js
Comment (Steve-Mcl) [inline frontend/src/api/devices.js:58]: Are these for future use Ben?

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (Steve-Mcl) [inline frontend/src/pages/device/components/DeviceLog.vue:55]:
```suggestion
```

Remove unused code

File: frontend/src/api/devices.js
Comment (hardillb) [reply frontend/src/api/devices.js:58]:
```suggestion
```

They should be obsolete now, the counter in the websocket code should handle all that now.

Comment (Steve-Mcl) [reply frontend/src/api/devices.js:58]: ok, which means the `exports` at lines 68,69 and the async call in https://github.com/flowforge/flowforge/blob/365bf107be8c8b46cc3c101924469fc44019fb13/frontend/src/pages/device/components/DeviceLog.vue#L52 should also be removed right?

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (hardillb) [inline frontend/src/pages/device/components/DeviceLog.vue:52]:
```suggestion
```

File: frontend/src/pages/device/components/DeviceLog.vue
Comment (hardillb) [inline frontend/src/pages/device/components/DeviceLog.vue:86]:
```suggestion
```

File: frontend/src/api/devices.js
Comment (hardillb) [inline frontend/src/api/devices.js:69]:
```suggestion
    updateSettings
```

File: frontend/src/pages/device/Logs.vue
Comment (Steve-Mcl) [reply frontend/src/pages/device/Logs.vue:6]: last one @hardillb 

Is it necessary to keep commented code?

Comment (hardillb) [reply frontend/src/pages/device/Logs.vue:6]: I don't think so, I removed them when talking to Pez about the lint changes. I'll commit this suggestion.

Please pull and re-test this before merging just to be sure

**Review by Steve-Mcl — APPROVED** [review]
All good for me.  Will merge once tests complete.

**Pull request merged**
